### DISCUSSION TITLE
Match directories based on different URLs

### DISCUSSION THREAD

**Discussion by FalconWu2017** [nginx/nginx#331] [General]
How to bind different URLs to different physical paths?for example:
`http://localhost/` bind  to  `d:\website1\`   
`http://localhost/dt` bind to `d:\website2\`   
The server_name must be the same.   
Any way?

**Comment by Maryna-f5** [comment]
This discussion appears to focus on the configuration, troubleshooting or general questions around running NGINX, which is not the intended use for GitHub discussions. 

We encourage you to visit [https://community.nginx.org ](https://community.nginx.org/)and share your discussion there as the NGINX Community Forum is the most appropriate place to engage for these types of questions.
